### ISSUE TITLE
label_image in view function does not show the segmented tumor!

### ISSUE THREAD

**Issue by ManooshSamiei** [kirbyju/TCIA_Notebooks#26]
Hello, 
I have tried all the notebooks in this repo, but i noticed in the new itk versions, i cant visualize the segmented tumor in the 3d visulization. Can you please help me find the issue?

**Comment by kirbyju** [comment]
I had some trouble installing itk stuff after upgrading to python 3.12.  Here's the issue I submitted and their reply: https://github.com/InsightSoftwareConsortium/ITK/issues/4451.  After downgrading to Python 3.11 things worked again with itk 3.5.0.  Not sure if this is your problem or if it's something else.  I'd need more details.  Also cc'ing @aylward as I've noticed that when I run through the MONAI notebook it only shows the prostate images and not the labels/segmentations.  Maybe that's also what @ManooshSamiei was trying to say?

**Comment by aylward** [comment]
For fastest attention, this should be posted to the itkWidgets issue tracker.  
https://github.com/InsightSoftwareConsortium/itkwidgets/issues

What version of jupyter lab/notebook are your running, what os, version of itkwidgets, etc.

**Comment by ManooshSamiei** [comment]
> Also cc'ing @aylward as I've noticed that when I run through the MONAI notebook it only shows the prostate images and not the labels/segmentations. Maybe that's also what @ManooshSamiei was trying to say?

Yes exactly the segmentation is not shown. A clear example is MONAI notebook I also attach a screen shot of what should be shown in pink but is not shown...  The python version in colab is 3.10.12, and the itk version is 5.3.0.
<img width="1312" alt="Screenshot 2024-02-11 at 7 38 03 PM" src="https://github.com/kirbyju/TCIA_Notebooks/assets/36368861/23f4a89f-a5bc-426d-ae1d-b85065c1b280">

**Comment by kirbyju** [comment]
@ManooshSamiei Thanks for creating https://github.com/InsightSoftwareConsortium/itkwidgets/issues/737.  I'll keep an eye on their reply in case I need to update the notebooks here.

**Comment by kirbyju** [comment]
I've inquired in https://github.com/InsightSoftwareConsortium/itkwidgets/issues/737 to see if there's anything I can do, but I think we're still waiting on the itkWidgets team.
